core/db/default/spree/refund_reason.rb

```diff
@@ -0,0 +1 @@
+Spree::RefundReason.find_or_create_by(name: "Return processing")
```

Comment:
I think this should be added [here](https://github.com/solidusio/solidus/blob/4354667e613ceeac29c4fc30c7fbc1e5abc56083/core/db/default/spree/return_reasons.rb) instead